CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 #### Changes
 * Python: Added JSON.DEL JSON.FORGET commands  ([#1146](https://github.com/aws/glide-for-redis/pull/1146))
-* 
+* Core: Added `BLPOP` and `BRPOP` commands ([#1218](https://github.com/aws/glide-for-redis/pull/1218))
```

Comment:
looks like we are just adding blpop to core, although we are obviously adding brpop as well to Java (just not to core)

Reply:
Right, but there is practice to put here complete PR names as is, but I'm open to edit it.
@barshaul what do you think?